### PR TITLE
Josir Gomes Bio

### LINKED ISSUE #1830: Onboarding for Josir Gomes
This is an onboarding checklist for @josircg

## Purpose of this checklist

This checklist is a GUIDELINES for use by anyone responsible for welcoming a new team member onto the project. It was originally written with lesson editors in mind. It therefore may need to be adjusted for people entering non-editing roles to ensure they have the training and support they need to become effective team members. Please use this in the spirit in which it was created rather than viewing it as a rigid set of rules.

---

## Welcome

Welcome to the Programming Historian Project Team. We're very pleased to have you on our team. We know that joining a new project and getting up to speed can be a challenge. So we've put together this information to help you get up to speed. It includes details of what we will do and a checklist of tasks you should complete, to make sure you know how to perform key tasks. We hope it won't take you very long and that this process will enable you to contribute your ideas and energy to their fullest.

## What we will do

Within 2 weeks of your joining, our current team will ensure that you have been granted access to the following services used by the Project Team:

- [x] Our Google Group email so you can communicate with other editors (@jenniferisasi)
- [x] Our Google Analytics account for monitoring web traffic (@acrymble)
- [x] Our Twitter team (@jenniferisasi)
- [x] Our Github Repository so you can upload materials to the site (@ZoeLeBlanc)
- [x] Our Skype group so that you can participate in our monthly calls (@mariajoafana )
- [ ] partner you with an existing editor to develop a 3 month plan so that you can acclimatise to the project.
- [x] invite you to become a 'member' of ProgHist Ltd, our not-for-profit company that administers the business end of the project.

## What you should do

There are a few tasks you will need to perform to make sure you know how to use our system, and also to make sure your details appear on the Project Team page. All of these tasks are encouraged, but some may be optional:

### communication

- [x] set up a GitHub account and inform @ZoeLeBlanc of your username (mandatory)
- [x] set up a twitter account (optional)
- [x] set up a Skype account and inform @mariajoafana (mandatory)
- [x] inform @acrymble and @jenniferisasi of your Google-activated email (mandatory)
- [x] write an email to the other editors using the Google group (say hello!) (mandatory)

### technical

We rely heavily on Github for communicating and conducting our editorial processes. You will need to get familiar with it and the way we use it. We recognise that this can take time, so if you would like one-on-one training, please let someone know and this will be arranged. To make sure you can do all required tasks, try the following activities on our Github /jekyll website.

- [x] contribute to one open ticket on our github repository (mandatory)
- [x] create and assign to yourself a new ticket on the github repository to ensure you know how to do this. Add a relevant label. (mandatory)
- [x] make a technical contribution on your own. This will normally be adding your bio to the /data/ph_authors.yml file, and uploading a square black and white avatar (.png) of yourself to /avatars). You will then submit a pull request for approval so that this bio information can be added to the Project Team page (mandatory)
- [x] Read all sections of the /contribute pages of the website, including author, reviewer, and editor guidelines
- [x] Bookmark the /jekyll (https://github.com/programminghistorian/jekyll) and /submissions (https://github.com/programminghistorian/ph-submissions) github sites so you can find them easily
- [x] Read the 'Making Technical Contributions' wiki page: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

### The Culture of the Project

We have invited you to join the team because we value your opinions. We encourage all of our editors to contribute to discussions on github or via email, and to civilly express their views, even if they disagree with any or all other members. This right to express one's opinions is central to our project. Please take this to heart.

### LINKED ISSUE #1838: Onboarding process: adding my bio to the repository
As requested in #1830 I am adding my bio information in /data/ph_authors.yml
Could not add myself as assignees...

### PR DESCRIPTION
**Pull request by josircg**
Closes #1838 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

File: _data/ph_authors.yml
Comment (rivaquiroga) [inline _data/ph_authors.yml]: in this case is team: true

File: _data/ph_authors.yml
Comment (rivaquiroga) [inline _data/ph_authors.yml:1374]: The Spanish translation is missing. I'll add it tomorrow. Whenever you need a translation, you can tag the team + the Managing Editor to request it (https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines)

**Comment by rivaquiroga** [comment]
Hi @josircg! Welcome to the team. I added the Spanish translation of your Bio + the missing team roles. 
Your affiliation is also missing. Let us know when you complete that, so we can add the translation.

**Comment by josircg** [comment]
Thank you @rivaquiroga . I will add the affiliation and photo on a second commit. 
I was anxious to test if the pull request requirements are ok :)

I was in doubt if I have to have travis installed on my machine to send the pull request so I did a small change to test. I could not assign myself to the issue and I thought this could be a problem either.

**Comment by walshbr** [comment]
@josircg - Travis is a web service that doesn't need to be installed on your computer. And you should be able to assign yourself to a pull request, but ultimately that shouldn't cause any errors if some quirk of the interface won't let you do so. It's more about keeping things organized and tidy.

**Comment by rivaquiroga** [comment]
@josircg, maybe you can't assign yourself because you are not yet part of the team here in Github. Did you inform Zoe about your github username?

**Comment by josircg** [comment]
No. I imagined that it was informed already. But I did it now.

**Comment by DanielAlvesLABDH** [comment]
Josir already mentioned his username in this message (https://github.com/programminghistorian/jekyll/issues/1830#issuecomment-656125578). Probably the problem was the opening of another issue.

**Comment by acrymble** [comment]
@josircg can you please use the checklist in the original comment above? This helps make sure you've done all the checks before someone reviews it. It's a series of reminders of how to do a proper pull request.

File: _data/ph_authors.yml
Comment (josircg) [reply _data/ph_authors.yml]: I understood that team was just for the main team. But I got it now.

**Comment by josircg** [comment]
> @josircg can you please use the checklist in the original comment above? This helps make sure you've done all the checks before someone reviews it. It's a series of reminders of how to do a proper pull request.

Hi. I don't have access yet to do any change on the original issues (#1830), or assign myself to them. I've already asked for help to @ZoeLeBlanc

**Comment by josircg** [comment]
Well... I just found the invitation on my spam box. And it's expired :/ 
@ZoeLeBlanc @rivaquiroga  Could you send another invitation?

**Comment by josircg** [comment]
@rivaquiroga I think everything is ok now. Need only your final approval.

**Comment by rivaquiroga** [comment]
@josircg, the french translation is missing. You need to tag the french team + Marie Christine to ask for the translation.

**Comment by josircg** [comment]
> @josircg, the french translation is missing. You need to tag the french team + Marie Christine to ask for the translation.

I asked someone else to translate but I'm tagging @mariechristineb to verify if it's ok.

File: _data/ph_authors.yml
Comment (mariechristineb) [inline _data/ph_authors.yml]: Josir Gomes est doctorant en sciences de l'information au PPGCI/IBICT/UFRJ. Diplômé en informatique, il se spécialise dans les systèmes d'information, les bases de données relationnelles, l'exploration de données et les logiciels libres.

**Review by mariechristineb — CHANGES_REQUESTED** [review]
I didn't want to start a new branch so I added my changes to the translation in a comment

File: _data/ph_authors.yml
Comment (josircg) [reply _data/ph_authors.yml]: Merci beacoup

**Comment by acrymble** [comment]
Did you include your photo in the pull request @josircg ?

**Review by JoshuaGOB — DISMISSED** [review]

**Comment by josircg** [comment]
> Did you include your photo in the pull request @josircg ?

No. I will do it in a second commit because I have to find/assemble a squared black/white photo and didn't want to late the onboarding process. Photo added now.

**Comment by rivaquiroga** [comment]
@josircg, it seems that the only thing missing is your affiliation.

**Comment by rivaquiroga** [comment]
Last thing and then we are ready to merge: the file name of your photo must be the same you use in `ph_authors`. If you want to appear as Josir Cardoso Gomes there, the file name should be `josir-cardoso-gomes.png`

**Review by rivaquiroga — APPROVED** [review]

**Comment by josircg** [comment]
Hi @rivaquiroga. Should I have to close the issue or is it closed only when deployed?

**Comment by JoshuaGOB** [comment]
Hi @josircg We're are just waiting for the review from @mariechristineb and then we can close the ticket. Is this correct @rivaquiroga ?

**Review by mariechristineb — APPROVED** [review]

**Pull request merged**
